library-app/src/App.module.css

```diff
@@ -0,0 +1,8 @@
+@media only screen and (min-width: 426px) {
+  .wrapp {
```

Comment:
Use more descriptive class name. This is good name if class is reusable and enough descriptive for it's use case.